internal/templatefn/fn.go

```diff
@@ -496,3 +486,20 @@ func lpaDecisions(app page.AppData, lpa any, canChange bool) lpaDecisionsData {
 
 	return data
 }
+
+func summaryRow(app page.AppData, label, value, changeLink, fullName string, optional, canChange, summarisingSelf bool) map[string]any {
```

Comment:
although that would be annoying for some places you've got, so maybe ignore this

Reply:
maybe having an `optional-summary-row` template would simplify things, since there are only a few places it is true. And the others mostly do `$canChange` so you can tell what arg that is

Reply:
Yeah fair point - I'll split this use case in to `optional-summary-row`